optlang/expression_parsing.py

```diff
@@ -70,6 +70,7 @@ def _parse_linear_expression(expression, expanded=False, **kwargs):
     """
     offset = 0
     constant = None
+    one = Integer(1)
```

Comment:
I don't know how much overhead this adds (probably very little), but I guess this could be moved outside of the function so it's only executed once when importing